src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public String getSalaryInfo(String[] names, String[] data,
+                                              String dateFrom, String dateTo)
+            throws DateTimeParseException {
+        int wagePerHour;
+        int workingHours;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
![image](https://github.com/mate-academy/jv-salary-info/assets/20326858/442d75dd-2cd7-4792-bcd0-c5e323767280)

Reply:
Variable made private final and brought outside method